src/realm/descriptor.hpp

```diff
@@ -326,6 +334,11 @@ class Descriptor : public std::enable_shared_from_this<Descriptor> {
     //@}
 
     //@{
+    /// Returns the contained spec.
+    Spec* get_spec() const;
+    //@}
```

Comment:
Why is it necessary to expose the `Spec` call in the public part of the API? While `Descriptor` is a public class, `Spec` is not, at least according to my original design. Note that we already have `_impl::DescriptorFriend::get_spec()` for internal use cases.